modules/ROOT/pages/java-embedded/traversal.adoc

```diff
@@ -63,9 +59,8 @@ At depth 3 => Agent Smith
 Number of friends found: 4
 ----
 
-Who coded the Matrix?
-To find the master mind behind `The Matrix`, we want to follow all outgoing relationships which
-have the type `KNOWS` or `CODED_BY`.
+=== Who coded the Matrix?
+To find the master mind behind "The Matrix", follow all outgoing relationships which have the type `KNOWS` or `CODED_BY`.
```

Comment:
This rhetorical question in the title looks a little odd - perhaps something like this as an alternative? (Also, mastermind is one word I think!)

```suggestion
=== Solving the Matrix

To find the mastermind behind "The Matrix", follow all outgoing relationships which have the type `KNOWS` or `CODED_BY`.
```